### ISSUE TITLE
how to get a finite element solution?

### ISSUE THREAD

**Issue by Qiang-Feng98** [paulpuren/seismic_modeling#2]
Hi, Thanks for your work.
I would like to know how to get a finite element solution. Is it obtained through simulation software? It would be appreciated if you could give more details about the real solution.

**Comment by paulpuren** [comment]
> Hi, Thanks for your work. I would like to know how to get a finite element solution. Is it obtained through simulation software? It would be appreciated if you could give more details about the real solution.

Yes, we use finite element solutions, which are simulated from Abacus. The details of numerical simulation are included in the paper. Thanks.
